src/styles/_header.scss

```diff
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: 300ms;
```

Comment:
add transition property in 40 line. By default, transition property is all, and it's hard for the browser to control all properties.

Reply:
add cursor: pointer for &__card